backend/tasks/user.py

```diff
@@ -0,0 +1,87 @@
+from flask_restful import Resource, reqparse
+from flask import request
+from psycopg2.extras import RealDictCursor
+from pymysql.cursors import DictCursor
+import os
+from flask import make_response
+import json
+import logging
+from flask_login import login_required
+logger = logging.getLogger(__name__)
+
+
+class UserTask(Resource):
+
+    @login_required
+    def get(self, user):
+        if user is None or len(user) == 0:
+            return make_response(json.dumps({'message': 'Parameter \'user\' is required'}), 400)
+
+        try:
+            with request.db.cursor(cursor_factory=RealDictCursor) if os.environ.get('IS_LOCAL') else request.db.cursor(DictCursor) as cur:
+                cur.execute(f"SELECT \"user\".username, \"user\".enabled FROM \"user\" WHERE \"user\".username='{user}'")
```

Comment:
I suspect username cannot contain symbols like `'`, but just to be safe it's better to write it as
```
cur.execute("SELECT \"user\".username, \"user\".enabled FROM \"user\" WHERE \"user\".username=%s", (user,))
```
or perhaps I would write it as
```
cur.execute("SELECT u.username, u.enabled FROM "user" u WHERE u.username=%s", (user,))
```

Reply:
You are correct, I added in the correct parameterization of passed parameters to the SQL queries.